src/main/java/org/apache/commons/compress/archivers/dump/DumpArchiveConstants.java

```diff
@@ -135,15 +135,32 @@ public static SEGMENT_TYPE find(final int code) {
     public static final int TP_SIZE = 1024;
 
     /**
-     * NTREC value {@value}.
+     * Minimum number of kilobytes per record: {@value}.
+     */
+    public static final int MIN_NTREC = 1;
+
+    /**
+     * Default number of kilobytes per record: {@value}.
      */
     public static final int NTREC = 10;
 
     /**
-     * HIGH_DENSITY_NTREC value {@value}.
+     * Number of kilobytes per record for high density tapes: {@value}.
      */
     public static final int HIGH_DENSITY_NTREC = 32;
 
+    /**
+     * Maximum number of kilobytes per record: {@value}.
+     * <p>
+     *     This limit matches the one used by the Linux ext2/3/4 dump/restore utilities.
+     *     For more details, see the
+     *     <a href="https://dump.sourceforge.io/">Linux dump/restore utilities documentation</a>
+     *     and the
+     *     <a href="https://manpages.debian.org/unstable/dump/dump.8.en.html#b,">dump(8) man page</a>.
+     * </p>
```

Comment:
New public and protected elements need a Javadoc `@since` tag. The question really is: Does this really need to be public or can it be package-private or private?

Reply:
Fixed in https://github.com/apache/commons-compress/pull/697/commits/9e21d6adec7d56cd6f9f411969e4643d483da890